src/Console/ModelsCommand.php

```diff
@@ -749,7 +749,7 @@ public function setProperty($name, $type = null, $read = null, $write = null, $c
         }
     }
 
-    protected function setMethod($name, $type = '', $arguments = [], $comment = '')
+    public function setMethod($name, $type = '', $arguments = [], $comment = '')
```

Comment:
As with https://github.com/barryvdh/laravel-ide-helper/pull/945#issuecomment-812680184 , this will bring the same kind of breaking change.

So maybe in this case we should bump the major?

Reply:
I don't know, I think the impact is pretty small. The intended use case is not to provide an extensible package for all methods imho.